types/index.d.ts

```diff
@@ -454,7 +469,10 @@ declare module "miragejs/server" {
     ): void;
 
     /** Pass through one or more URLs to make real requests. */
-    passthrough(urls?: ((request: Request) => any) | string | string[]): void;
+    passthrough(...urls: PassthroughUrl[]): void;
+    passthrough(
+      ...args: [PassthroughUrl, ...PassthroughUrl[], PassthroughVerbs]
+    ): void;
```

Comment:
Here we have new types for pass through, its either just urls, or at least one url followed by array of verbs.